hudi-integ-test/src/main/java/org/apache/hudi/integ/testsuite/dag/nodes/RollbackNode.java

```diff
@@ -49,6 +54,11 @@ public void execute(ExecutionContext executionContext) throws Exception {
     Option<HoodieInstant> lastInstant = metaClient.getActiveTimeline().getCommitsTimeline().lastInstant();
     if (lastInstant.isPresent()) {
```

Comment:
looks like our rollback only supports rolling back the last commit. I assume we need to fix this in some later patch.

Reply:
So, you can add multiple rollbackNode's to simulate multiple commit rollbacks, it will always start from the last node. We still have to implement a RestoreNode that will do this in one shot..

Reply:
my bad. I missed that we can rollback only the latest commit.